K2Bridge/Factories/BucketFactory.cs

```diff
@@ -23,16 +23,32 @@ public static DateHistogramBucket CreateFromDataRow(DataRow row)
         {
             Ensure.IsNotNull(row, nameof(row));
 
+            // TODO: timestamp is always the first row (probably named _2), and count will be named _count
```

Comment:
can you add a link to the issue?

Reply:
https://dev.azure.com/CSECodeHub/476026%20-%20K2Bridge%20OSS%20-%20Visualization%20Capabilities/_workitems/edit/15050

Reply:
Link to the issue added to code.